dv/uvm/core_ibex/Makefile

```diff
@@ -248,8 +244,24 @@ instr-gen-build-vars-prereq = \
 # Finally, $(instr-gen-build-vars-prereq) becomes a dependency of our target.
 ################## END EXAMPLE ###################
 
+core-config-vars-path := $(BUILD-DIR)/.cc.vars.mk
+-include $(core-config-vars-path)
+
+core-config-var-prereq = $(call vars-prereq,gen,Generate core configuration file,$(core-config-var-deps))
+$(CORE-CONFIG-STAMP): \
+	$(core-config-var-prereq) ./riscv_dv_extension/riscv_core_setting.tpl.sv \
+	scripts/render_config_template.py \
+	| $(BUILD-DIR)
+	@echo Generating core configuration file
+	$(verb)env PYTHONPATH=$(PYTHONPATH) \
+	scripts/render_config_template.py \
+	  --dir-metadata $(METADATA-DIR) \
+		$(EXT_DIR)/riscv_core_setting.tpl.sv > $(EXT_DIR)/riscv_core_setting.sv
```

Comment:
We've also been doing a single tab plus spaces for alignment for lines of a recipe.